code/modules/mob/living/carbon/human/human.dm

```diff
@@ -73,6 +73,9 @@
 /mob/living/carbon/human/tajaran/Initialize(mapload)
 	..(mapload, /datum/species/tajaran)
 
+/mob/living/carbon/human/vulpkanin
+	name += "McCloud"

```

Comment:
This wont work due to the way speech is handled in the code. Please look at modifying the say proc instead of mob default names, as this will be overrode on character spawning.

Reply:
Went for initlize. May look at say in a bit.